t/plugin/batch-requests.t

```diff
@@ -988,7 +988,7 @@ passed
 GET /t
 --- error_code: 400
 --- response_body eval
-qr/\{"error_msg":"invalid configuration: property \\"max_body_size\\" validation failed: expected 0 to be sctrictly greater than 0"\}/
+qr/\{"error_msg":"invalid configuration: property \\"max_body_size\\" validation failed: expected 0 to be strictly greater than 0"\}/
```

Comment:
I think we need to update the source code and release a new version:

https://github.com/api7/jsonschema/search?q=sctrictly

Reply:
I created a new issue: https://github.com/api7/jsonschema/issues/51

Reply:
The new version is not released yet.

@jbampton 
Can you submit a PR like https://github.com/api7/jsonschema/pull/45 so that we can release a new version?